packages/embed/src/factories/create-sidetab/create-sidetab.ts

```diff
@@ -1,4 +1,5 @@
 import { createIframe, setElementSize, handleCustomOpen, unmountElement, setAutoClose } from '../../utils'
+import { addCustomKeyboardListener } from '../../utils/'
```

Comment:
Why is this import separated from above? The trailing slash doesn't seem necessary and it could be in the other utils import declaration

Reply:
Fixing it